src/style.css

```diff
@@ -1 +1,123 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
```

Comment:
Be consistent with your margins. It's recommended to add margins only to the top or bottom of elements, not both, to avoid potential margin collapse. In this case, you're adding margin to all sides of the `body` element.